### PR TITLE
Add location from EXIF data

### PR DESCRIPTION
**Pull request by cadnce**
Added location extraction from exif data.
Very occasionally works, majority of twitter images don't have geolocation embedded.

### CODE REVIEW

File: lambdas/extractTweetGeo/tweet_utils.py [hunk 1]
```diff
@@ -72,7 +105,7 @@ def extract_location(doc):
 
     try:
         if doc['coordinates']:
-            return doc['coordiantes']
+            return doc['coordinates']
```
Comment (cadnce) [inline lambdas/extractTweetGeo/tweet_utils.py:108]: So I'd actually fixed this previously in what was running in "prod"

**Comment by stevenjfr** [comment]
2AM?!?

**Comment by cadnce** [comment]
U mad bro?

**Review by forseti7 — APPROVED** [review]
Making sound progress here Matt, keep up the good work.

**Pull request merged**